### ISSUE TITLE
explain telemetry better

### ISSUE THREAD

**Issue by Thorin-Oakenpants** [arkenfox/user.js#1660]
> then why do we disable it in AF

this is something I have mulled over for years - i.e it is somewhat at odds with trying to keep the mission on statement - e.g see removal of most personal section items. Also see how we still enable Safe Browsing except the real-time binary checks. The "problem" is the user base: they just want a "quiet" fox and if I excluded them or made them inactive, then it would just bloat most user's overrides.

That said - I should get around to adding some context, maybe in a new section. A bit like the intro to SB

_Originally posted by @Thorin-Oakenpants in https://github.com/arkenfox/user.js/issues/1659#issuecomment-1518353397_

**Comment by Thorin-Oakenpants** [comment]
true, we can probably make that line inactive and keep it for the info value

**Comment by worstflimsy** [comment]
Main reason I disable telemetry or atleast minimize it is to prevent telemetry. Since I'm already commenting here, that means I change prefs - just few. But I don't want software developers -  Mozilla in this case - what I'm doing with their software. If I encounter any issue I would first confirm it in default settings and then report it if it shows its face again. That takes BUT TELEMETRY IS IMPORTANT TO REPORT ISSUES TO DEVELOPERS out. Also if telemetry is not disabled, don't be surprised to see some additional data sent to servers in future updates which will happen by accident as explanatory statement will say afterward. But we know the real reason behind it. Things happen, even unexpected ones. And even if we all love Firefox, Mozilla has made mistakes before. Just my two outdated cents.

**Comment by Thorin-Oakenpants** [comment]
just to be CRYSTAL CLEAR .. this is not about anyone's preferences or assumptions. I only deal in absolutes. If anyone is expecting to air perceived grievances or transgressions or talk about not trusting Mozilla, then expect a swift ban

To be EVEN MOAR CLEAR ... disabling telemetry will stay - I simply want to differentiate it from privacy/tracking issues. Telemetry is not a privacy issue - I am not talking about experiments (which is a subset and almost exclusively nightly anyway)

**Comment by worstflimsy** [comment]
Okay well no more Mozilla grievances although I use Firefox 99% of the time and happy with it, which means Mozilla must be doing something right.

I like telemetry where data which is sent is displayed before uploading. This is different from initial connection any software makes on startup which usually are for proper functioning and security of browser. That's why current approach of this project is fine. Essential connections allowed which can't be disabled in any software which is crucial for proper functioning of browser. Telemetry is disabled and if there are issues, check whether they are present in default settings and then report it to Mozilla. There's a reason this approach held its own for so long. However an internal review is good to weigh pros and cons for a change, in this case telemetry.

Even if there are no tracking issues in telemetry, having more controls or atleast mention of it in future versions to change that yourself at your own risk, if these settings do change in this project would be good.

**Comment by Thorin-Oakenpants** [comment]
you can view all telemetry data in about:telemetry and auto-displaying it before sending would be a terrible user experience. The rest makes no sense just like the previous comment

THIS IS NOT A DISCUSSION ABOUT TELEMETRY ... this is an issue to explore adding some extra info to AF to differentiate it from actual privacy issues, maybe even give it it's own section, so it makes sense as to _why_ it's in AF

**Comment by Thorin-Oakenpants** [comment]
> Let everyone decide for themselves

That's the whole point of a template :)

**Comment by Thorin-Oakenpants** [comment]
that's the whole point - we set default hard'ish then they find their own way. If they can't read a wiki page with 10 basic prefs in it to help them, then they shouldn't be here

**Comment by Thorin-Oakenpants** [comment]
one of the points - https://hacks.mozilla.org/2018/10/testing-privacy-preserving-telemetry-with-prio/

**Comment by adrelanos** [comment]
Ticket re-titeling suggested:

> Move Telemetry to its own Section and Improve Telemetry Disabling Settings Documentation

Because currently `do something about telemetry` sounds as if there are unresolved issues with telemetry disabling, which  is not what this ticket is about.

**Comment by Thorin-Oakenpants** [comment]
@souchikjoardar201 No. No. and No. What makes you think any of these are a privacy or security risk.

> without breakage

yeah, right

Disabling
- push - pointless
   - it requires a subscription and it is E2EE, so not a privacy issue
   - it alters your fingerprint (from almost all the other FF users), for no reason
   - it is required for CRLite - so in fact you are adding a security issue
   - creates friction and breakage for users
- updates
   - as a default I will NEVER disable auto-updating (of anything) for security and **_compat_** (breakage) )reasons
   - users can always change the setting via the UI if they want to go down this road, but I am not going to even provide the pref as a FYI because this is just an idiotic idea and I do not want to promote it
- geo
   - it requires permission
   - it creates friction and breakage for users
   - it alters your fingerprint (from almost all the other FF users), for no reason

read the godamn user js - two of these three are listed in `[SECTION 7000]: DON'T BOTHER` with the reasons `WHY` you should leave them the fuck alone

**Comment by Thorin-Oakenpants** [comment]
We could probably add, thanks to @Obegg in #1845 that sanitizing `profile\datareporting` is a good idea since some pending pings of whatnot may still linger and cause a DNS lookup - I mean the point is "quiet" fox, so lets help do that if people want it

**Comment by Thorin-Oakenpants** [comment]
https://github.com/arkenfox/user.js/pull/1929/commits/4f37c32a0e219cc7deefff33b5683f546e3d8ac9

@rusty-snake and others - feels like I've written a book - anyway, comments welcome to improve my wording - the aim is to dispel nonsense that others claim elsewhere - ergo. "it should be opt-in","telemetry is spyware", "Firefox is tracking you and selling your web traffic data" blah blah blah - seen many people claim it doesn't even prompt you to opt-out (it does, not exactly an in-your-face popup - prompt you have to deal with - it's an new tab loaded in a new profile's first use)

anyways - closing, but comment welcome to improve what I'm trying to say

**Comment by ghost** [comment]
Push services should be disabled:

```
user_pref("dom.push.enabled", false);
user_pref("dom.push.server.URL", "");
user_pref("dom.push.userAgentID", "");
```

**Comment by xeleven11** [comment]
`toolkit.telemetry.enabled` why is this still locked to true in recent Firefox. I tried looking for answers but no one explains what this does and how it can be disabled

**Comment by Thorin-Oakenpants** [comment]
RTFM

https://github.com/arkenfox/user.js/blob/0f14e030b3a9391e761c03ce3c260730a78a4db6/user.js#L1239
